engine/src/main/resources/assets/ui/worldSetupScreen.ui

```diff
@@ -83,6 +83,30 @@
                   }
                 ]
               },
+              {
+                "type": "RowLayout",
+                "id": "custom_name",
```

Comment:
Generally avoid `_` in naming and use camel case styling instead (so `customName`) - but ideally be more descriptive like `worldName` or `customWorldName`